bands/models.py

```diff
@@ -18,81 +18,111 @@ class Event:
 class Band(MagModel):
     group_id = Column(UUID, ForeignKey('group.id'))
     event_id = Column(UUID, ForeignKey('event.id'), nullable=True)
+    info = relationship('BandInfo', backref='band', uselist=False)
+    bio = relationship('BandBio', backref='band', uselist=False)
+    taxes = relationship('BandTaxes', backref='band', uselist=False)
+    stage_plot = relationship('BandStagePlot', backref='band', uselist=False)
+    panel = relationship('BandPanel', backref='band', uselist=False)
+    merch = relationship('BandMerch', backref='band', uselist=False)
+    charity = relationship('BandCharity', backref='band', uselist=False)
 
     payment = Column(Integer, default=0, admin_only=True)
-    vehicles = Column(Integer, default=1, admin_only=True)
-    estimated_loadin_minutes = Column(Integer, default=60, admin_only=True)
-    estimated_performance_minutes = Column(Integer, default=60, admin_only=True)
 
+    @property
+    def all_badges_claimed(self):
+        return not any(a.is_unassigned for a in self.group.attendees)
+
+    @property
```

Comment:
It was not especially helpful, it was just throwing an `AttributeError` on `BandInfo.estimated_performer_count`. It's possible that something went awry in actually saving/updating the code (which can happen when you make a lot of changes in a row), so I'll have another go before I call this truly broken.